colossalai/inference/pipeline/batch_infer_state.py

```diff
@@ -0,0 +1,120 @@
+# might want to consider combine with InferenceConfig in colossalai/ppinference/inference_config.py later
```

Comment:
Can we directly import from tensor_parallel here?

Reply:
done